modules/default/weather/current.njk

```diff
@@ -9,7 +9,11 @@
                 {% if config.useBeaufort %}
                     {{ current.beaufortWindSpeed() | round }}
                 {% else %}
-                    {{ current.windSpeed | round }}
+                    {% if config.useKmh %}
```

Comment:
I would find it nicer if you made that switch in the js code and just pass the windspeed here like before. That way the calculation (logic) stays in the js and display-related stuff stays in the njk.

Reply:
Ok i will try to revert changes in njk and implement modification in js.

Reply:
According to your suggestion, I make some changes, see commit 1460f002ab79c0aa209e56a44a41b91291d26d8e

(I don't know how to change the commit reference in this pull request. Previous commit is no longer necessary.)